examples/minimal-mdns/client.cpp

```diff
@@ -309,14 +309,15 @@ int main(int argc, char ** args)
 
     mdns::Minimal::Server<20> mdnsServer;
     ReportDelegate reporter;
+    CHIP_ERROR err;
```

Comment:
why this change? This increases the scope of err

Reply:
just trying to fix the build